Packaging/Debian/libdispatch-5.1.5/debian/control

```diff
@@ -0,0 +1,31 @@
+Source: libdispatch
+Priority: extra
+Maintainer: Mark Heily <mark@heily.com>
```

Comment:
I guess maintainer should be you. libdispatch 5.1.5 comes from vanilla Apple sources.

Reply:
I'd love to see this package updated in Debian officially, and I can help with it too. Where would i get the libdispatch-5.1.5.tar.gz ?

Reply:
download-origs.sh fetches the tarballs. for libdispatch it uses the tag swift-5.1.5-RELEASE from https://github.com/apple/swift-corelibs-libdispatch